app/controllers/markets_controller.rb

```diff
@@ -0,0 +1,46 @@
+class MarketsController < ApplicationController
+  before_action :get_market, only: [:edit, :show, :update]
+
+  def get_market
```

Comment:
Since this method is only used by the `before_action` I think it would make sense for it to be in the `private` section